<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnigdhaOSAssistant</class>
 <widget class="QMainWindow" name="SnigdhaOSAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Snigdha OS Assistant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="mainStackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="textWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QDialogButtonBox" name="textWidget_buttonBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
          </property>
          <property name="centerButtons">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QStackedWidget" name="textStackedWidget">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="textWidget_welcome">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Welcome to &lt;b&gt;Snigdha OS!&lt;/b&gt;ðŸ˜Š&lt;/span&gt;&lt;br/&gt;&lt;b&gt;Snigdha OS Assistant&lt;/b&gt; will help you set up your system to your wishes.&lt;/p&gt;&lt;p&gt;Before the Snigdha OS Assistant can continue, initial system updates have to be applied and the Snigdha OS Assistant is updated.&lt;br/&gt;Make sure you have a working internet connection before you continue.&lt;/p&gt;&lt;/br&gt;This application is inspired from &lt;b&gt;Garuda Linux&lt;/b&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textWidget_updateRetry">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Update failed!ðŸ˜’&lt;/span&gt;&lt;br/&gt;System update failed. Try again?ðŸ˜¬&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://forum.snigdhaos.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Visit the Snigdha OS Forum&lt;/span&gt;&lt;/a&gt; if further issues occour.&lt;br/&gt;Finishing this update via the Snigdha OS Assistant is highly recommended in order to apply release-specific fixes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textWidget_nvidia">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;NVIDIA drivers&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Additional nonfree NVIDIA graphics drivers are available. Do you want to install them?&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textWidget_applyRetry">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Failed to ApplyðŸ˜’!&lt;/span&gt;&lt;br/&gt;Applying system changes failed. Try againðŸ˜„?&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://forum.snigdhaos.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Visit the Snigdha OS Forum&lt;/span&gt;&lt;/a&gt; if further issues occour.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textWidget_success">
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Success!ðŸ˜Ž&lt;/span&gt;&lt;br/&gt;Your system is set up and ready to go!&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://forum.snigdhais.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Visit the Snigdha OS Forum&lt;/span&gt;&lt;/a&gt; if issues occour.&lt;br/&gt;&lt;a href=&quot;https://snigdhaos.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Snigdha OS Documentation&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://wiki.archlinux.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Arch Linux Wiki&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="textWidget_quit">
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Before you goðŸ§¸&lt;/span&gt;&lt;/p&gt;&lt;p&gt;You can restart the Snigdha OS Assistant at any time.&lt;br/&gt;Finishing the Snigdha OS Assistant is recommended in order to apply release-specific fixes.&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;https://forum.snigdhaos.org&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Visit the Snigdha OS Forum&lt;/span&gt;&lt;/a&gt; if issues occour.&lt;br/&gt;&lt;a href=&quot;https://snigdhaos.org/documentation/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Snigdha OS Documentation&lt;br/&gt;&lt;/span&gt;&lt;/a&gt;&lt;a href=&quot;https://wiki.archlinux.org/&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#754ffe;&quot;&gt;Arch Linux Wiki&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="textFormat">
               <enum>Qt::TextFormat::RichText</enum>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="waitingWidget">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="waitingWidget_text">
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="selectWidget">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QTabWidget" name="selectWidget_tabs">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <property name="elideMode">
           <enum>Qt::TextElideMode::ElideNone</enum>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>OS preferences</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_10">
            <item row="15" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>344</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="14" column="0">
             <widget class="QCheckBox" name="checkBox_Blackarch">
              <property name="text">
               <string>Do you need Pentesting software? (installs BlackArch repo + settings)</string>
              </property>
              <property name="packages" stdset="0">
               <stringlist notr="true">
                <string>blackarch-keyring</string>
                <string>blackarch-menus</string>
                <string>blackarch-mirrorlist</string>
               </stringlist>
              </property>
              <property name="prepare_commands" stdset="0">
               <stringlist notr="true">
                <string>sh &lt;(wget -qO- https://blackarch.org/strap.sh)</string>
               </stringlist>
              </property>
              <property name="setup_commands" stdset="0">
               <stringlist notr="true">
                <string>sed -i 's/#server/server/g' /etc/pacman.d/blackarch-mirrorlist</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QCheckBox" name="checkBox_Wallpaper">
              <property name="text">
               <string>Do you want to install additional Snigdha OS wallpapers?</string>
              </property>
              <property name="packages" stdset="0">
               <stringlist notr="true">
                <string>snigdhaos-additional-wallpapers</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="checkBox_GNOME">
              <property name="text">
               <string>Do you want to install additional GNOME applications?</string>
              </property>
              <property name="packages" stdset="0">
               <stringlist notr="true">
                <string>nautilus-image-converter</string>
                <string>nautilus-share</string>
                <string>nautilus-sendto</string>
                <string>eog-plugins</string>
                <string>grilo-plugins</string>
                <string>gnome-logs</string>
                <string>gnome-sound-recorder</string>
                <string>gnome-user-share</string>
                <string>lollypop</string>
                <string>celluloid</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBox_KDE">
              <property name="text">
               <string>Do you want to install additional KDE components and applications?</string>
              </property>
              <property name="packages" stdset="0">
               <stringlist notr="true">
                <string>appmenu-gtk-module</string>
                <string>ark</string>
                <string>bluedevil</string>
                <string>breeze</string>
                <string>breeze-gtk</string>
                <string>colord-kde</string>
                <string>dolphin-plugins</string>
                <string>drkonqi</string>
                <string>filelight</string>
                <string>ffmpegthumbs</string>
                <string>gwenview</string>
                <string>icoutils</string>
                <string>kaccounts-providers</string>
                <string>kactivitymanagerd</string>
                <string>kamera</string>
                <string>kamoso</string>
                <string>kate</string>
                <string>kcalc</string>
                <string>kcron</string>
                <string>kde-cli-tools</string>
                <string>kde-gtk-config</string>
                <string>kde-service-menu-reimage</string>
                <string>kde-servicemenus-encfs</string>
                <string>kde-servicemenus-komparemenu</string>
                <string>kde-servicemenus-pdf</string>
                <string>kde-servicemenus-pdf-encrypt-decrypt</string>
                <string>kde-servicemenus-officeconverter</string>
                <string>kde-servicemenus-sendtodesktop</string>
                <string>kde-servicemenus-setaswallpaper</string>
                <string>kdeconnect</string>
                <string>kdecoration</string>
                <string>kdegraphics-thumbnailers</string>
                <string>kdeplasma-addons</string>
                <string>kdf</string>
                <string>kdialog</string>
                <string>keditbookmarks</string>
                <string>kfind</string>
                <string>kgamma5</string>
                <string>khelpcenter</string>
                <string>khotkeys</string>
                <string>kimageformats</string>
                <string>kinfocenter</string>
                <string>kio-extras</string>
                <string>kio-fuse</string>
                <string>kio-gdrive</string>
                <string>kleopatra</string>
                <string>kmenuedit</string>
                <string>kompare</string>
                <string>konsole</string>
                <string>krdc</string>
                <string>krename</string>
                <string>krfb</string>
                <string>kscreen</string>
                <string>ksshaskpass</string>
                <string>ksystemlog</string>
                <string>kwalletmanager</string>
                <string>kwrited</string>
                <string>milou</string>
                <string>okular</string>
                <string>partitionmanager</string>
                <string>plasma-browser-integration</string>
                <string>plasma-desktop</string>
                <string>plasma-disks</string>
                <string>plasma-firewall</string>
                <string>plasma-integration</string>
                <string>plasma-nm</string>
                <string>plasma-pa</string>
                <string>plasma-systemmonitor</string>
                <string>plasma-thunderbolt</string>
                <string>plasma-vault</string>
                <string>plasma-workspace</string>
                <string>plasma-workspace-wallpapers</string>
                <string>polkit-kde-agent</string>
                <string>powerdevil</string>
                <string>qt5-imageformats</string>
                <string>quota-tools</string>
                <string>resvg</string>
                <string>rootactions-servicemenu</string>
                <string>ruby</string>
                <string>spectacle</string>
                <string>systemsettings</string>
                <string>yakuake</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QCheckBox" name="checkBox_Performance">
              <property name="text">
               <string>Do you want to apply additional performance tweaks? (at the cost of power usage/heat)</string>
              </property>
              <property name="packages" stdset="0">
               <stringlist notr="true">
                <string>performance-tweaks</string>
               </stringlist>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>344</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="0">
             <widget class="QCheckBox" name="checkBox_Samba">
              <property name="text">
               <string>Do you need Printer, Scanner and Samba Support?</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QDialogButtonBox" name="selectWidget_buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
